README.md

```diff
@@ -1,39 +1,36 @@
-# `f0cal/farm`
+# `f0cal/farm-client`
 
-Python API client and CLI wrapper for the FØCAL [device farm](https://f0cal.com/devices). 
+Python API client and CLI wrapper for the [FØCAL](https://f0cal.com) device farm API.
+
+Implements the `f0cal.farm.client` Python namespace.
 
 ## Quick start
 
+```bash
+pip install f0cal.farm.client \
+  -i https://dl.f0cal.com/py
+  
+f0cal farm --help
 ```
-git clone https://github.com/f0cal/farm && \
-  cd farm && \
-  pip install ./ -r requirements.txt -c constraints.txt && \
-  f0cal farm --help
-```
-## Basics
-
-FØCAL's [device farm](https://f0cal.com/devices) is "AWS for devices." This package is the device farm's canonical API client implementation. You can:
-
-* Spin up a device with the OS of your choice.
-* Log in to the device as a superuser.
-* Modify the device filesystem to meet your needs.
-* Save the modified device image for later.
-* Share the modified device image with collaborators.
```

Comment:
Not sure on where the above changes came from. Did you modify this or is this a merge from a different branch?

Reply:
I haven't made any changes to the readme, must've been merged in from this commit https://github.com/f0cal/farm-client/commit/c61f8b2a76dd09c2875287b97a3088e246aae882#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5